packages/roosterjs-editor-api/lib/utils/commitListChains.ts

```diff
@@ -11,7 +11,9 @@ export default function commitListChains(editor: IEditor, chains: VListChain[])
         const range = editor.getSelectionRange();
         const start = range && Position.getStart(range);
         const end = range && Position.getEnd(range);
-        chains.forEach(chain => chain.commit());
+        chains.forEach(chain =>
+            chain.commit(editor.isFeatureEnabled(ExperimentalFeatures.ReuseAllAncestorListElements))
```

Comment:
let's move the call to `editor.isFeatureEnabled(ExperimentalFeatures.ReuseAllAncestorListElements)` out of the loop